### DISCUSSION TITLE
IMPORTANT - v129-N/A skiovox unpatch

### DISCUSSION THREAD

**Discussion by akabutnicer** [3kh0/ext-remover#1272] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
hey guys, i've came to ask for help on a pending skiovox unpatch (which to be fair is unlike me)

currently, I've found a method to open and access a skiovox window in the background. although, simple methods like `alert()` doesn't seem to update its focus. with my new method you may or may not be able to access the extensions you had previously installed via the initial skiovox release.  so come together (I'm aware this is cringe but stay with me) and find a way to get this to work. i shall list the steps to this new method, so you all can experiment with processes. don't be retarded and ask what this exploit is, or how you're having trouble launching a kiosk app; we don't care.

steps:
1. signout (obviously)
2. connect to a network which requires authentication through a captive portal (i.g xfinity)
3. continue with the regular skiovox instructions (more specifically the kiosk launching part)
5. you should be on an error screen, click diagnose >  the first dropdown (should be named, 'wifi') > the settings link.
6. click TTS (text-to-speech) > check chromevox switch > click chromevox tutorial
7. click resources > any link
---
**now this is the hard part (if you can try and summarize this)**

continued steps:

8. press `tab` twice > press `search+space` > press `ctrl+a+backspace|delete` then type any link which may help you
9. open the dev console (`ctrl+shift+j`) then type in any script, to test i recommend `alert()`

 
assuming we did all the steps right, chromevox should tell you "alert"  from the page chromevox considers focused. this is where  you come in to try and find a way to actually get this page to focus. remember, you have NO WIFI, but you should also have access to the extensions you had previously installed.

**Comment by schoolexploitkid** [comment]
you can turn on wifi toggling in chrome://network, which will enable wifi.

**Reply by akabutnicer** [reply]
yeah fair but that only adds more steps, which isn't necessarily bad but I'm kind of against it

**Reply by akabutnicer** [reply]
in addition, i'm fairly sure this can be done without internet

**Comment by S-PScripts** [comment]
closed as these discussions are dead and you're in the relocate server
